### PR TITLE
Api genres

### PR DESCRIPTION
**Pull request by anetaszynal**

### CODE REVIEW

File: src/features/genres/genresSaga.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import {call, put, takeEvery} from "redux-saga/effects";
+import {getGenre} from "./getGenre";
+import {fetchGenreSucces, fetchGenreError, fetchGenre} from "./genresSlice";
+
+function* fetchGenreHandler() {
+    try {
+        const genre = yield call(getGenre);
+        yield put(fetchGenreSucces(genre));
+        console.log(genre)
```
Comment (damdlani) [inline src/features/genres/genresSaga.js]: You missed it ;)

File: src/features/genres/genresSlice.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    name: "genres",
+    initialState: {
+        isLoading: false,
+        genres: [],
+    },
+    reducers: {
+        fetchGenre: (state) => {
+            state.isLoading = true;
+        },
+        fetchGenreSucces: (state, {payload}) => {
```
Comment (damdlani) [inline src/features/genres/genresSlice.js:13]: Typo in name, you're missing 's' in the end, in each success reducer

**Review by damdlani — APPROVED** [review]
Excellent!

File: src/features/genres/genresSlice.js (hunk 2 above)
Comment (anetaszynal) [reply src/features/genres/genresSlice.js]: yes, thank you ;)

File: src/features/genres/genresSaga.js (hunk 1 above)
Comment (anetaszynal) [reply src/features/genres/genresSaga.js]: fixed :)

**Pull request merged**